proptools/extend_test.go

```diff
@@ -1451,9 +1506,11 @@ func TestExtendMatchingProperties(t *testing.T) {
 func check(t *testing.T, testType, testString string,
 	got interface{}, err error,
 	expected interface{}, expectedErr error) {
+	t.Helper()
 
 	printedTestCase := false
 	e := func(s string, expected, got interface{}) {
+		t.Helper()
```

Comment:
This function is not being passed t, does it still need the t.Helper designation?

Reply:
Yes, any function that is called multiple times and contains a t.Error method should use t.Helper() so that the test failure points to the line that called the helper function.